reactioncommerce/admin
# chore: adding git hooks for commit-msg

**Testing**
1. Pull the branch.
2. Run npm i
3. Make some edit.
4. Try to commit, something weird like "this is a invalid commit". The hooks should fail.
5. Fix the messge to something like "fix: valid commit", you should see the commit succeed.
